include/mbedtls/ssl_cookie.h

```diff
@@ -53,16 +53,23 @@ extern "C" {
  */
 typedef struct mbedtls_ssl_cookie_ctx
 {
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    mbedtls_svc_key_id_t    MBEDTLS_PRIVATE(psa_hmac);   /*!< key id for the HMAC portion   */
```

Comment:
Minor suggestion: naming it `psa_hmac_key` would be, in my opinion, cleaner.

Reply:
Thanks, indeed the name is better, I'll change it